app/Http/Controllers/BlogPostsController.php

```diff
@@ -44,7 +44,7 @@ public function store()
         $post->body = $data['body'];
         $post->save();
 
-        return redirect('/home');
+        return redirect('/profile');
```

Comment:
If possible, please change this also. And sorry, I forgot to mention but I believe you can remove the 「URL:: part」 and just go with 「route('profile')」. Can you check if its possible? If it throws an error, then no need to change and ignore my other comments for now.